data_nutch_data_nci/conf/nutch-site.xml

```diff
@@ -63,19 +63,23 @@
 
 	<property>
 		<name>http.timeout</name>
-		<value>10000</value>
+		<value>20000</value>
 		<description>NCI OVERRIDE: Upped to 20sec from 10sec to allow time for www.cancer.gov to generate its sitemap.xml file. 20151113 - it seems 20sec is not long enough, so reverting to 10sec</description>
 	</property>
 	
+
+
+
+
 	<property>
 	  <name>fetcher.threads.fetch</name>
-	  <value>100</value>
+	  <value>50</value>
 	  <description>NCI OVERRIDE: Upping this to 100</description>
```

Comment:
More disagreement between the configuration and the adjoining comments.

Reply:
updated the comment to use 50